--- a/modules/audio_output/apple/coreaudio_common.c
+++ b/modules/audio_output/apple/coreaudio_common.c
@@ -26,7 +26,7 @@
 #include "channel_layout.h"
 #include <CoreAudio/CoreAudioTypes.h>
 
-#define TIMING_REPORT_DELAY_TICKS VLC_TICK_FROM_MS(1000)
+#define TIMING_REPORT_DELAY_TICKS VLC_TICK_FROM_MS(100)
 
 static inline uint64_t
 BytesToFrames(struct aout_sys_common *p_sys, size_t i_bytes)
@@ -269,7 +269,7 @@
     {
         p_sys->timing_report_last_written_bytes = 0;
 
-        /* From now on, fetch the timestamp every 1 seconds */
+        /* From now on, fetch the timestamp every 100 milliseconds */
         p_sys->timing_report_delay_bytes =
             TicksToBytes(p_sys, TIMING_REPORT_DELAY_TICKS);
 
